### PR TITLE
Define the block size less variable in js

### PR DESCRIPTION
**Pull request by frostburn**
Only set block swap style when necessary

### CODE REVIEW

File: ui/fancy/src/js/variables.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+/* eslint-env browser */
+
+// This module mirrors variables.less
+
+const blockSize = [2.5, 'em'];
```
Comment (RauliL) [inline ui/fancy/src/js/variables.js]: You can use `module.exports.blockSize = ` instead.

Comment (RauliL) [inline ui/fancy/src/js/variables.js]: I'm not that happy about using array for this, an object with properties `name` and `unit` would be more readable.

Comment (frostburn) [inline ui/fancy/src/js/variables.js]: I guess you're right.

**Comment by frostburn** [comment]
Fixed readability.

**Pull request merged**